spec/models/qing_group_spec.rb

```diff
@@ -0,0 +1,14 @@
+require "spec_helper"
+
+describe QingGroup do
+  context "One QingGroup" do
+    before do
+      @form = FactoryGirl.create(:sample_form, question_types: [['text', 'text', 'text']])
```

Comment:
The sample_form factory is deprecated. You can just use regular `:form`. Could you actually put a comment on it? It was a bad idea.